docs/user-guide/import.md

```diff
@@ -16,10 +16,20 @@ Here the user, in order to import a file, can drag and drop it inside the import
 
 ## Import map context files
 
-A map context is, for example, the file that an user download selecting the <img src="../img/button/export.jpg" class="ms-docbutton"/> option in [Burger Menu](menu-bar.md#burger-menu). This type of files, in `json` format contains all map context information, that means the settings about projections, coordinates, zoom, extent and layers present in the map (more information can be found in the [Maps Configuration](../developer-guide/maps-configuration.md) section of the [Developer Guide](https://mapstore.readthedocs.io/en/latest/developer-guide/). Adding a map configuration file the behavior is similar to the following:
+A map context is, for example, the file that an user download selecting the <img src="../img/button/export.jpg" class="ms-docbutton"/> option in [Burger Menu](menu-bar.md#burger-menu). There are two types of file that can be exported:
+
+* The <img src="../img/button/mapstore_format_button.jpg" class="ms-docbutton"/> file, is a `json` format contains all map context information, that means the settings about projections, coordinates, zoom, extent and layers present in the map (more information can be found in the [Maps Configuration](../developer-guide/maps-configuration.md) section of the [Developer Guide](https://mapstore.readthedocs.io/en/latest/developer-guide/).
+
+Adding a MapStore configuration file the behavior is similar to the following:
 
 <img src="../img/import/export-import.gif" class="ms-docimage"/>
 
+* The <img src="../img/button/wmc_format_button.jpg" class="ms-docbutton"/> (Web Map Context) file, is a `xml` format that contains only the layers present in the map and keeping their settings related to projections, coordinates, zoom and extension (more information can be found in the  [Maps Configuration](../developer-guide/maps-configuration.md#web-map-context) section of the [Developer Guide](https://mapstore.readthedocs.io/en/latest/developer-guide/).
```

Comment:
The <img src="../img/button/wmc_format_button.jpg" class="ms-docbutton"/> (Web Map Context) file, is a `xml` format where only WMS layers present in the map are exported including their settings related to projections, coordinates, zoom and extension (more information can be found in the  [Maps Configuration](../developer-guide/maps-configuration.md#web-map-context) section of the [Developer Guide](https://mapstore.readthedocs.io/en/latest/developer-guide/).